README.md

```diff
@@ -1,2 +1,2 @@
-# product-dev
+# product-developement-magic
```

Comment:
@gauravds here is a typo issue with `development`

Reply:
I fixed that, please check it again